docs/pages/reference/cli.mdx

```diff
@@ -1973,6 +1973,40 @@ $ tctl status \
     --identity=identity.pem
 ```
 
+### tctl alerts create
+
+Creates cluster alerts
+
+```$ code
+$ tctl alerts create [<flags>] <message>
+```
+
+#### Arguments
+
+- `<message>`: The message for the alert to display
+
+#### Flags
+
+| Name | Default Value(s) | Allowed Value(s) | Description |
+| - | - | - | - |
+| `--severity` | `low` | `low`, `medium`, `high` | Alert severity |
+| `--ttl` | none | Any duration | Optional expiry for alert (alert does not expire if no TTL is specified) |
+| `--labels` | none | any | A list of labels to attach to the alert. Labels can be used to tweak alert behavior |
```

Comment:
- What are some quick examples of ways that labels can be used to tweak alert behavior?

- How would a user determine what values to supply for `--labels`? Are these the same labels that are already assigned to cluster resources, or something else?

Reply:
That's covered down below. There are currently two labels (`alert-on-login` and `alert-permit-all`) that affect alert behavior.